ddtrace/internal/_tagset.pyx

```diff
@@ -152,7 +152,7 @@ cdef bint _value_is_valid(str value):
         return 0
 
     for c in value:
-        if not is_valid_key_char(ord(c)):
+        if not is_valid_value_char(ord(c)):
```

Comment:
why do we need to change this method?

Reply:
it is a small bug, catching at the same time since they are related.

Reply:
I can move this to it's own PR if people want. I'm not sure if I'd backport it so felt maybe it was ok?